projects/whydense/gsc/experiments.cfg

```diff
@@ -132,3 +132,22 @@ log_interval = 400
 batches_in_epoch = 5121
 batch_size = 16
 
+; Apply k-winners across the channels
+[SuperSparseCNNLocal]
```

Comment:
Would be best to move this into exps.cfg (or another cfg file) until we have a good version that would go into the paper.